apps/cow-fi/app/(main)/legal/cowswap-terms/page.tsx

```diff
@@ -697,44 +696,111 @@ export default function Page() {
+              retroactive reward for past activity.
             </p>
-            <h3 id="base-launch-giveaway-period">3. Giveaway Period</h3>
-            <p>From 2024-12-12 at the time of the post on the X Platform to 2024-12-31, at 11:59 PM UTC.</p>
-            <h3 id="base-launch-how-to-enter">4. How to Enter</h3>
+            <h3 id="base-token-reward-eligibility">2. Eligibility</h3>
             <p>
-              Trade on Base using CoW Swap during the Giveaway Period. Quote Tweet the Giveaway announcement with your
-              trade's transaction hash. One entry per eligible transaction.
+              Must be a real user who conducted trades on CoW Swap on the Base Network during February 2025 and must:
             </p>
-            <h3 id="base-launch-prize">5. Prize</h3>
+            <ul>
+              <li>be minimum 18 years of age.</li>
+              <li>Not be an employee, affiliate, or immediate family member of the Organizer.</li>
+            </ul>
+            <h3 id="base-token-reward-period">3. Reward Period</h3>
             <p>
-              10,000 COW tokens distributed among randomly selected winners. The Organiser will determine the number of
-              winners and token allocation at its discretion.
+              This reward program is based on trading activity that occurred during the period from{' '}
+              <strong>February 1st, 2025 to February 28th, 2025</strong> (the “Period”) on the Base Network using CoW
+              Swap. This is a retroactive program and not an ongoing or future giveaway.
             </p>
-            <h3 id="base-launch-winner-selection">6. Winner Selection</h3>
-            <p>Random selection from eligible entries. Odds depend on the number of entries.</p>
-            <h3 id="base-launch-winner-notification">7. Winner Notification</h3>
-            <p>Winners announced on X within 15 days after the Giveaway Period.</p>
-            <h3 id="base-launch-publicblockchain">8. Public Blockchain</h3>
+            <h3 id="base-token-reward-how-to-enter">4. How to be Selected & Reward</h3>
+            <ul>
+              <li>
+                <strong>No action is required from users to be considered for this reward.</strong> Users who traded on
+                CoW Swap on the Base network in the Period are automatically considered for this retroactive reward.
+              </li>
+              <li>
+                Three (3) users will be <strong>randomly selected</strong> from all users who traded on CoW Swap on Base
+                during Period.
+              </li>
+              <li>
+                Each of the three (3) selected users will receive a reward of <strong>1000 COW tokens.</strong>
+              </li>
+              <li>
+                The reward will be sent to the wallet address used to trade on CoW Swap on base during the Period.
+              </li>
```

Comment:
<li>
                The reward will be sent to the wallet address used to trade on CoW Swap on Base during the Period.
              </li>